src/d_deh.c

```diff
@@ -43,61 +44,82 @@ static boolean bfgcells_modified = false;
 // (e.g. from wads)
 
 typedef struct {
-  byte *inp, *lump; // Pointer to string or FILE
-  long size;
+  MEMFILE *stream;
+  FILE *file;
 } DEHFILE;
 
 // killough 10/98: emulate IO whether input really comes from a file or not
 
 // haleyjd: got rid of macros for MSCV
 
-char *dehfgets(char *buf, size_t n, DEHFILE *fp)
+char *dehfgets(char *str, size_t count, DEHFILE *fp)
```

Comment:
I guess we can turn all these functions into `static`?

Reply:
Done. BTW we can add `static` to a lot of functions, should I do it?